File: Day 45/DSA/Check_if_a_String_Is_an_Acronym_of_Words.java
Question:-  https://leetcode.com/problems/check-if-a-string-is-an-acronym-of-words/

Code:- 

class Solution {
    public boolean isAcronym(List<String> words, String s) {
        StringBuilder strbld = new StringBuilder();

        for(int i = 0;i<words.size();i++)
        {
            strbld.append(words.get(i).charAt(0));
        }

        return s.equals(strbld.toString());
    }
}

File: Day 45/DSA/Check_if_a_String_is_an_acronym_of_words.cpp
Question:-  https://leetcode.com/problems/check-if-a-string-is-an-acronym-of-words/





class Solution {
public:
    bool isAcronym(vector<string>& words, string s) {
        string str  = "";

        for(int i = 0;i<words.size();i++)
        {
            str += words[i][0];
        }

        return str == s;
    }
};